server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/documents.ts

```diff
@@ -52,6 +52,75 @@ import { Constants, getSession, StageTrace } from "../../../utils";
 import { IDocumentDeleteService } from "../../services";
 import type { RequestHandler } from "express-serve-static-core";
 
+/**
+ * Response body shape for modern clients that can handle object responses.
+ * @internal
+ */
+interface ICreateDocumentResponseBody {
+	/**
+	 * The id of the created document.
+	 */
+	id: string;
+	/**
+	 * The access token for the created document.
+	 * TODO: This is getting generated multiple times. We should generate it once and reuse it.
+	 */
+	token?: string;
```

Comment:
What does it mean for this to be undefined? Can we document that? Same with `session` below.